src/base/bittorrent/torrenthandle.h

```diff
@@ -282,6 +285,9 @@ namespace BitTorrent
         virtual void setUploadLimit(int limit) = 0;
         virtual void setDownloadLimit(int limit) = 0;
         virtual void setSuperSeeding(bool enable) = 0;
+        virtual void setDisableDHT(bool enable) = 0;
```

Comment:
Inappropriate parameter name, don't you think?

Reply:
Indeed :)